src/participant/prompts/index.ts

```diff
@@ -26,12 +27,18 @@ export class Prompts {
     for (const message of messages) {
       if (
         message.role === vscode.LanguageModelChatMessageRole.User &&
-        message.content.trim().length > 0
+        !isContentEmpty(message)
       ) {
         return true;
       }
     }
 
     return false;
   }
+
+  public static getContentLength(
```

Comment:
Can we just export-import in this case? The function does not introduce any logic.